devicetypes/smartthings/zwave-sound-sensor.src/zwave-sound-sensor.groovy

```diff
@@ -40,7 +40,13 @@ metadata {
 def installed() {
 	sendCheckIntervalEvent()
 	sendEvent(name: "sound", value: "not detected", displayed: false, isStateChanged: true)
-	response(zwave.batteryV1.batteryGet().format())
+	response([
+			zwave.batteryV1.batteryGet().format(),
+			"delay 2000",
+			zwave.wakeUpV1.wakeUpIntervalSet(seconds:14400, nodeid: zwaveHubNodeId).format(), // sometimes the 4hr wakeup interval is not durable
```

Comment:
Try without this set. Once you told me that the NoMoreInformation seemed to make a difference I was skeptical that this was the problem.